scholarx/archive/functions.js

```diff
@@ -263,40 +249,79 @@ function filterByYear() {
     const selectedIndustries = selectedIndustriesData();
     const selectedUniversities = selectedUniversitiesData();
     //helps to achieve mentees page cofunctionality with cohort filters 
-    if(document.getElementById("selection").value === "mentees"){
-        if(selectedUniversities.length == 0 && selectedYears.length == 0){
+    if (document.getElementById("selection").value === "mentees") {
+        if (selectedUniversities.length == 0 && selectedYears.length == 0) {
             renderAllProfiles()
             return
         }
         if (selectedUniversities.length == 0) {
-           filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
-           filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
-           renderProfiles(filteredMentors, filteredMentees);
-        } else{
+            filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
+            filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
+            renderProfiles(filteredMentors, filteredMentees);
+        } else {
             filterByUniversity("")
         }
         return;
     }//mentors page cofunctionality
-    if(selectedIndustries.length == 0 && selectedYears.length == 0){
+    if (selectedIndustries.length == 0 && selectedYears.length == 0) {
         renderAllProfiles()
         return
     }
     if (selectedIndustries.length == 0) {
-       filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
-       filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
-       renderProfiles(filteredMentors, filteredMentees);
-    } else{
+        filteredMentors = mentors.filter((mentor) => selectedYears.includes(mentor.year));
+        filteredMentees = mentees.filter((mentee) => selectedYears.includes(mentee.year));
+        renderProfiles(filteredMentors, filteredMentees);
+    } else {
         filterByIndustry("")
     }
 }
-function openMentorProfile(index){
+//mentor mentee selection
+function mentorsCheckout() {
+    var mentorsCheckbox = document.getElementById('mentorsCheckbox');
+    mentorsCheckbox.addEventListener('change', function () {
+        if (mentorsCheckbox.checked) {
+            uncheckCheckboxes();
+            renderAllProfiles();
+            $("#showMentors").show();
+            $("#showMentees").hide();
+            $("#university-filter").hide();
+            $("#industry-filter").show();
+        }
+    });
+}
```

Comment:
```suggestion
function mentorsCheckout() {
            uncheckCheckboxes();
            renderAllProfiles();
            $("#showMentors").show();
            $("#showMentees").hide();
            $("#university-filter").hide();
            $("#industry-filter").show();
}
```

What happen if you write the code this way?

Reply:
Yeah its also working I'll update like this @janithlahirukariyawasam